### PR TITLE
Various UI tweaks

### PR DESCRIPTION
**Pull request by frederoni**

### CODE REVIEW

File: MapboxNavigation/RouteMapViewController.swift [hunk 1]
```diff
@@ -45,8 +45,8 @@ class RouteMapViewController: UIViewController {
     var tiltedCamera: MGLMapCamera {
         get {
             let camera = mapView.camera
-            camera.altitude = 600
-            camera.pitch = 45
+            camera.altitude = 150
+            camera.pitch = 70
```
Comment (bsudekum) [inline MapboxNavigation/RouteMapViewController.swift]: I think 60 might be the max.

**Comment by bsudekum** [comment]
@frederoni mind posting a screenshot?

File: MapboxNavigation/RouteMapViewController.swift (hunk 1 above)
Comment (ericrwolfe) [inline MapboxNavigation/RouteMapViewController.swift]: Also, what are the implications of this on battery life @pveugen?

Comment (pveugen) [inline MapboxNavigation/RouteMapViewController.swift]: Didn't do any pre- and post testing, but anecdotally this does have some impact.

File: MapboxNavigation/RouteTableViewController.swift [hunk 2]
```diff
@@ -42,6 +42,8 @@ class RouteTableViewController: UIViewController {
             headerView.distanceRemaining.text = distanceFormatter.string(from: routeProgress.distanceRemaining)
         }
         
+        dateComponentsFormatter.unitsStyle = routeProgress.durationRemaining < 3600 ? .short : .abbreviated
```
Comment (1ec5) [inline MapboxNavigation/RouteTableViewController.swift]: I think `dateComponentsFormatter` rounds up when the duration is close to an hour. So there might be an edge case where it still uses `short` even though you want `abbreviated`.

File: MapboxNavigation/RouteTableViewController.swift [hunk 3]
```diff
@@ -20,8 +20,8 @@ class RouteTableViewController: UIViewController {
         setupTableView()
         dateFormatter.timeStyle = .short
         dateComponentsFormatter.maximumUnitCount = 2
```
Comment (1ec5) [inline MapboxNavigation/RouteTableViewController.swift]: This is no longer necessary because `day` has been removed below.

**Comment by frederoni** [comment]
Superseded by https://github.com/mapbox/mapbox-navigation-ios/pull/480

**Pull request closed without merging**